istio-telemetry/kiali/templates/demosecret.yaml

```diff
@@ -0,0 +1,16 @@
+{{- if .Values.kiali.createDemoSecret }}
```

Comment:
I would remove this from the new installer. 
Maybe we can use kustomize or a script to setup the secret - but it seems a bad practice to checkin any secret or use hardcoded.

Reply:
I disagree, it is for demo usage, to require people to make a secret is an unneeded step. Unless we can use kustomize and require no user steps somehow

Reply:
it is only for demo purpose, I'd vote for removing it only if we can find an equivalent experience and step and have it documented.

Reply:
might keep it for now as it is demo purpose only